docs/continuous_replication.md

```diff
@@ -18,6 +18,7 @@ If you lose your main pool, you can start the copy on the other side, with very
 
 :::warning
 It is normal that you can't boot the copied VM directly: we protect it. The normal workflow is to make a clone and then work on it.
+This also affects VMs with "Auto Power On" enabled, because of our protections you can ensure these won't start on your CR destination if you happen to reboot it.
```

Comment:
Markdown does not respect line break, if you want this sentence to stand out, add an empty line before to make it a paragraph.

Reply:
fixed for Yannick :)